src/app/explore/explore.component.ts

```diff
@@ -124,6 +130,16 @@ export class ExploreComponent implements OnDestroy, OnInit {
   }
 
   onScroll() {
+    if (this.scroller.getScrollPosition()[1] < this.smallestSignificantScrollPx) {
+      this.blurredSinceLastMajorScroll = false;
```

Comment:
@bdsl The idea is that if you go back to the top, you are more likely to be about to interact with the search fields. So there's a reasonable chance that it will be useful to blur afresh when you're done. I'll add a comment.